### PR TITLE
Add support for Radio Kyivstar

### PR DESCRIPTION
**Pull request by chugylo**
Support for https://radio.kyivstar.ua

See how I normalized the URL. Should I move this snippet to the core?

### CODE REVIEW

File: src/connectors/kyivstar.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Connector.playButtonSelector = '.play';
+
+Connector.trackArtSelector = '#song_image';
+
+Connector.getTrackArt = () => {
+	let trackArtUrl = $(Connector.trackArtSelector).attr('src');
+
+	// Normalize the URL here because the protocol cannot be resolved in the background script what
+	// leads to absent picture in popups and notifications.
+	if (trackArtUrl.startsWith('//')) {
```
Comment (inverse) [inline src/connectors/kyivstar.js]: Like you mentioned this could def be moved to the core :+1:

File: src/connectors/kyivstar.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	for (let file of defaultFiles) {
+		if (`${location.origin}/img/appimg/${file}` === trackArtUrl) {
+			return true;
+		}
+	}
+
+	return false;
+};
+
+Connector.isScrobblingAllowed = () => Connector.getArtist() !== 'Kyivstar';
```
Comment (inverse) [inline src/connectors/kyivstar.js]: What is `Kyivstar`?

Comment (chugylo) [reply src/connectors/kyivstar.js]: From time to time, the radiostation make small annoncements and this label is changing to Kyivstar, the name of the company.

**Comment by inverse** [comment]
Thanks @chugylo :+1:

**Pull request merged**